dialect.go

```diff
@@ -4,7 +4,7 @@ import "time"
 
 // Dialect abstracts database differences
 type Dialect interface {
-	QuoteIdent(id string) string
+	QuoteIndent(id string) string
```

Comment:
this is not type. Ident - is short form from Identifier.
Dialect is public interface and it need to keep it without change.

Reply:
hm... if you try to go implements, you see next:

```go
func (d clickhouse) QuoteIndent(s string) string {
	return quoteIndent(s, "`")
}
```

```go
func (d mysql) QuoteIndent(s string) string {
	return quoteIndent(s, "`")
}
```

```go
func (d postgreSQL) QuoteIndent(s string) string {
	return quoteIndent(s, `"`)
}
```

```go
func (d sqlite3) QuoteIndent(s string) string {
	return quoteIndent(s, `"`)
}
```

Reply:
@bgaifullin this is not `Identifier`, it's `quote indent`

Reply:
I still insist on the mistake

Reply:
I see that all implementations are updated as well. But the purpose of this method is quote identifier, not indent ))

Reply:
`identifier` == `id`
`indent` == `"` or "\`" or `'`

Reply:
isn't it?

Reply:
identifier is column name or table name or something else that identifies a entity in DB
indent is indentation.

Reply:
hm..

Reply:
reverted